src/Tools/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Codefix.Testing/CodeFixTest`1.cs

```diff
@@ -248,6 +262,37 @@ protected async Task VerifyFixAsync(CancellationToken cancellationToken)
                     throw new Exception($"file name was expected to be '{newSources[i].filename}' but was '{updatedDocuments[i].Name}'");
                 }
             }
+
+            var updatedAdditionalDocuments = project.AdditionalDocuments.ToArray();
+
+            if (fixedAdditionalFiles.Length != updatedAdditionalDocuments.Length)
+            {
+                throw new Exception($"expected '{nameof(fixedAdditionalFiles)}' and '{nameof(updatedAdditionalDocuments)}' to be equal but '{nameof(fixedAdditionalFiles)}' contains '{fixedAdditionalFiles.Length}' documents and '{nameof(updatedAdditionalDocuments)}' contains '{updatedAdditionalDocuments.Length}' documents");
```

Comment:
Can we use the `TVerifier` class for these instead of exceptions?

Reply:
➡️ I used the pattern from the existing code for the new code. I'll submit a separate change to move from exceptions to `Verify`.